requirements.txt

```diff
@@ -1,17 +0,0 @@
-django >= 1.11.5
-elasticsearch==5.3.0
-enum==0.4.6
-jsonschema==2.6.0
-Markdown==2.6.5
-pymongo==3.2
-pyOpenSSL >=17.1.0,<18.0
-python-dateutil==2.4.2
-pytz==2015.7
-requests==2.9.1
-rfc3987==1.3.4
-strict-rfc3339==0.7
-unicodecsv==0.9.0
-uwsgi==2.0.13.1
-webapp2==2.5.2
-WebOb==1.5.1
-git+https://github.com/flywheel-io/gears.git@v0.1.4#egg=gears
```

Comment:
I use this file for [CI](https://github.com/flywheel-io/sdk/blob/3e5dd6ff31b6edc652215753f7865151d2f44b59/.circleci/config.yml#L31), so I would need an equivalent command. I'll want to try things out with this branch if it gets closer to merging.

Reply:
We will make sure that you don't need any of that anymore. You should able to simply stand up a core and use it. No internal knowledge required.

Reply:
Sounds good, as long as the container checks itself on boot. Frequently I want to CI against a branch whose container (incl updated libraries) has not built yet.

Reply:
That's a good point. We'll need to have images available for arbitrary branches of core to facilitate testing of interacting repos, such as reaper and cli. We typically build images for master and all tags. I think that's enough (rather than building for all branches) and keeps clutter at bay. Would that work for you, @kofalt?

Reply:
Discussed this out of band - basically, as long as any CI container can update itself on boot to the tip of a branch (incl updated libraries) we're fine.